src/palace/manager/api/admin/controller/patron.py

```diff
@@ -48,9 +52,29 @@ def _load_patron_data(
             )
 
         for provider in patron_lookup_providers:
-            remote_patron_data = provider.remote_patron_lookup(patron_data)
-            if remote_patron_data:
-                return remote_patron_data
+            try:
+                remote_patron_data = provider.remote_patron_lookup(patron_data)
+                if isinstance(remote_patron_data, ProblemDetail):
+                    return remote_patron_data
+                if remote_patron_data:
+                    return remote_patron_data
```

Comment:
It looks like this can be simplified:

```suggestion
                if remote_patron_data:
                    return remote_patron_data
```